zookeeper-server/src/main/java/org/apache/zookeeper/cli/SetCommand.java

```diff
@@ -38,10 +39,11 @@ public class SetCommand extends CliCommand {
     static {
         options.addOption("s", false, "stats");
         options.addOption("v", true, "version");
+        options.addOption("b", false, "base64");
     }
 
     public SetCommand() {
-        super("set", "[-s] [-v version] path data");
+        super("set", "[-s] [-v version] [-b] path data");
```

Comment:
No hex option here?

Reply:
Yes, hexdump only provided to reading binary data, since it's output in multiline, it is not appropriate support directly in single line command here.

Below is a hexdump output example:
```
get -x /foo
         +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 7a 6f 6f 6b 65 65 70 65 72                      |zookeeper       |
+--------+-------------------------------------------------+----------------+
```